client/my-sites/themes/theme-showcase.jsx

```diff
@@ -215,6 +223,7 @@ class ThemeShowcase extends React.Component {
 				) }
 				<div className="themes__content">
 					<QueryThemeFilters />
+					<ShowcaseBanner />
```

Comment:
I think it makes sense to just inline `<RandomThemesBanner context="theme-showcase" banners={ themeBanners } />` here, no need for the extra level of indirection (i.e. the `ShowcaseBanner` component).

Reply:
(You could also inline `[ PhotoBlogBanner, SmallBusinessBanner ]` directly into the `banners` prop.)

Reply:
Yeah, you're right. Since it's a function, it's the same to be specified inline as well. Good point.